### DISCUSSION TITLE
Can you tell me what extension is used for the plus function on the left?

### DISCUSSION THREAD

**Discussion by Yakumo-Yukari** [ueberdosis/tiptap#5967] [General Questions] [answered]
![image](https://github.com/user-attachments/assets/d281fa2f-44e3-4ba5-ac46-b08db18b8fd3)

**Comment by nperez0111** [comment, marked as answer]
The drag-handle extension allows for arbitrary content within the drag-handle. So we put a menu inside of it to allow inserting of new content, like notion does.
